lib/retriable.rb

```diff
@@ -6,15 +6,20 @@
 module Retriable
   module_function
 
-  def self.configure
+  def configure
```

Comment:
@kamui i don't know if you want to grab this change.. the `self` is gratuitous because of the `module_function`